lib/opentelemetry_plug.ex

```diff
@@ -103,21 +137,25 @@ defmodule OpentelemetryPlug do
   defp optional_attributes(conn) do
     # for some reason Elixir removed Enum.filter_map in 1.5
     # so just using Erlang's list module
```

Comment:
You could use `Enum.flat_map/2`. Is the concern efficiency?

Reply:
Kinda doubt it; 𝑛=2. Just another tell that the author comes more from the Erlang side of things? I'd flat map with empty lists instead of false, yeah: this part of the diff is just `mix format` doing its thing while I edit elsewhere.